ovos_config/locale.py

```diff
@@ -10,19 +10,24 @@
 except ImportError:
     LF = None
 
-_lang = "en-us"
+_lang = None
 _default_tz = None
 
 
 def get_primary_lang_code():
     if LF:
         return LF.get_primary_lang_code()
+    if not _lang:
+        _lang = ovos_config.Configuration().get("lang", "en-us")
```

Comment:
Does `Configuration` need a cast to `dict` for `get` to call the correct method here?

Reply:
no, Configuration() is a dict itself and fully compatible

only Configuration  (class not instance!) uses the "old get"